frontend/src/components/worksheets/ConfigPanel/ConfigPanel.jsx

```diff
@@ -0,0 +1,85 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import { withStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+
+class ConfigurationPanel extends React.Component<{
```

Comment:
Would be good to have a bit more high-level documentation about what the design principles, and where information goes where.